packages/data-service/src/native-client.js

```diff
@@ -1334,11 +1330,11 @@ function addDebugToClass(cls) {
     }
 
     const orig = descriptor.value;
-    descriptor.value = function(...args) {
+    descriptor.value = function (...args) {
       debug(`${prop}()`, args);
-      if (args.length > 0 & typeof args[args.length - 1] === 'function') {
+      if ((args.length > 0) & (typeof args[args.length - 1] === 'function')) {
```

Comment:
I wonder if this was supposed to be `&&` instead of `&` all this time

Reply:
Yes should be `&&` - anything else doesn't make sense here. I've quickly tried it, behavior for booleans seems to be exactly the same comparing `&` and `&&`.

Reply:
Yeah, I don't even really suggest changing it (although we totally can and it wouldn't hurt), just an interesting side-effect of running prettier against the code 😄